instances/atlas/parcellationEntity/AAL1/AAL1_ACIN.jsonld

```diff
@@ -0,0 +1,27 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/AAL1_ACIN",
+  "@type": "https://openminds.ebrains.eu/sands/ParcellationEntity",
+  "abbreviation": "ACIN",
+  "alternateName": [
+    "CIA",
+    "Cingulum_Ant"
+  ],
+  "definition": null,
+  "hasParent": [
+    {
+      "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/AAL1_limbicLobe"
+    }
+  ],
+  "hasVersion": [
+    {
+      "@id": "https://openminds.ebrains.eu/instances/parcellationEntityVersion/AAL1_SPM12-v4_ACIN"
```

Comment:
This is check to see that I understand the AAL1 correctly:

As this is a PE for a cingulate structure, the only version that contain a cingulate structure is the AAL1_SPM12 (called AAL 3: New version of anterior cingulate, thalamus and brain nuclei (nucleus accumbens, substantia nigra, ventral tegmental area, red nucleus, locus coeruleus, and raphe nuclei) on the webpage).

The 'hasParent' entry looks logical but I don't know where to check that it is true.

Otherwise, this looks good and it seems that the automatic generation was good

Reply:
@aeidi89 To be honest: I do not yet fully understand the AALs. Nonetheless, the AAL1 includes the cingulate structures as well (according to the paper and according to the data). For the moment I left out the mismatches between paper and data, to investigate further.